lib/maglev/listeners/broadcaster.rb

```diff
@@ -186,7 +186,12 @@ def broadcast_listeners
           end
 
           if !listened and event.targets.any?
-            raise EventError.new("event target was specified for #{listener.class.name} that does not support the #{event.name} event")
+            error = "event target was specified for #{listener.class.name} that does not support the #{event.name} event"
+            if MagLev.production?
+              warn error
```

Comment:
Where does warn come from? Isn't it `logger.warn error`?

Reply:
Cool I'll switch this, I just went with the Kernel/warn